go.mod

```diff
@@ -1,4 +1,4 @@
-go 1.15
+go 1.17
```

Comment:
changing version in this file means that go 1.17 is the minimum version required to run the SDK, right?

I don't think that's desirable then, would prefer to revert this file.

Reply:
Why is it not desirable? Elaborate, please.

Reply:
related https://github.com/cosmos/cosmos-sdk/pull/9588#discussion_r659582289

Reply:
Ohh this change was done manually, yeah? If that is the case then yes, I agree with you.

Reply:
should we bump it to 1.16 then?

Reply:
> should we bump it to 1.16 then?

Well, considering that go 1.15 is no longer supported I'd say yes, we could. The `go` tool should bump it automatically when required though.

Reply:
Yeah 1.16 sgtm too